lib/libthr/thread/thr_syscalls.c

```diff
@@ -641,6 +641,20 @@ __thr_interpose_libc(void)
 {
 
 	__set_error_selector(__error_threaded);
+#define	SLOT(name)					\
+	*(__libc_interposing_slot(INTERPOS_##name)) =	\
+	    (interpos_func_t)__thr_##name;
+	SLOT(system);
+	SLOT(tcdrain);
+	SLOT(spinlock);
+	SLOT(spinunlock);
+	SLOT(map_stacks_exec);
+#undef SLOT
+
+	/*
+	 * XXX: should be __libsys_interposing_slot once enough time has
```

Comment:
I do not think we need to wait to start using __libsys_interposing_slot() in libthr.  We do require that rtld/libc/libthr where updated atomically.

Reply:
OK, as long as we install libc before libthr (as noted in https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=197366#c8) we can do that. I think I need to keep there fallback in `__libc_interposing_slot` though.